src/OrchardCore/OrchardCore.Abstractions/Json/JOptions.cs

```diff
@@ -62,8 +62,8 @@ static JOptions()
 
         Document = new JsonDocumentOptions
         {
-            CommentHandling = Default.ReadCommentHandling,
-            AllowTrailingCommas = Default.AllowTrailingCommas,
+            CommentHandling = JsonCommentHandling.Skip,
+            AllowTrailingCommas = true,
```

Comment:
There is no value change here. I just did not like how we are depending on the Default options.